corehq/form_processor/models.py

```diff
@@ -1518,7 +1518,7 @@ class FormEditRebuild(CaseTransactionDetail):
 
 
 class FormReprocessRebuild(CaseTransactionDetail):
-    _type = CaseTransaction.TYPE_REBUILD_FORM_EDIT
+    _type = CaseTransaction.TYPE_REBUILD_FORM_REPROCESS
```

Comment:
Don't have much context here so trying to understand this.
how would that effect what has been happening till now?

Reply:
There is no impact - the result of this is simply that the transaction type is incorrect (32 instead of 2048) which is slightly confusing when looking at the transaction history. There is no logic built around these rebuild types though.

Reply:
okay cool,
thinking if in future someone is looking at a form and thinking if this was from a reprocess, but it looks it was from an edit. Just thinking about us choosing not to fix the data, probably we even can't now since there wont be a way to know it now.

Reply:
You can still tell by looking at the json structure of `transaction.details`:
```
# reprocess
{"form_id":"15008f0a-c943-4403-8e5c-9cfbcfda7755"}

# form edit
{"deprecated_form_id":"d66adff9-9382-4f62-b7dd-21f73cbe8dbb"}
```